Solutions/SemperisDirectoryServicesProtector/Workbooks/Images/Logo/Semperis.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 73.6 98.2"><defs><linearGradient id="a" y1="27.34" x2="73.5" y2="27.34" gradientTransform="matrix(1, 0, 0, -1, 0, 99.84)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ee4036"/><stop offset="0.19" stop-color="#ee3c3c"/><stop offset="0.43" stop-color="#ed2f4c"/><stop offset="0.71" stop-color="#ec1b67"/><stop offset="1" stop-color="#eb008b"/></linearGradient><linearGradient id="b" x1="0" y1="74.09" x2="73.49" y2="74.09" xlink:href="#a"/><linearGradient id="c" x1="36.85" y1="75.55" x2="73.6" y2="75.55" gradientUnits="userSpaceOnUse"><stop offset="0"/><stop offset="1" stop-color="#fff"/></linearGradient><linearGradient id="d" x1="36.85" y1="42.75" x2="73.5" y2="42.75" xlink:href="#c"/><linearGradient id="e" x1="36.85" y1="16.36" x2="73.6" y2="16.36" xlink:href="#c"/></defs><g style="isolation:isolate"><path d="M70.8,58.6l-34-5.7L.1,46.8v8.5a3.22,3.22,0,0,0,2.7,3.2l34.1,5.8,23.9,4a2,2,0,0,1,1.6,1.9V76a1.92,1.92,0,0,1-1.3,1.8L37.7,86a2.2,2.2,0,0,1-.8.1A2.77,2.77,0,0,1,36,86L12.5,77.8A1.92,1.92,0,0,1,11.2,76l.1-8.5L.1,65.6V82.2a3.22,3.22,0,0,0,2.1,3L35.9,98a3.55,3.55,0,0,0,1,.2,2.92,2.92,0,0,0,.9-.2L71.5,85.2a3.22,3.22,0,0,0,2.1-3V61.8A3.51,3.51,0,0,0,70.8,58.6Z" style="fill:url(#a)"/><path d="M2.7,39.7l34.1,5.7,36.7,6.1V43a3.22,3.22,0,0,0-2.7-3.2L36.8,34,12.9,30a2,2,0,0,1-1.6-1.9V22.3a1.92,1.92,0,0,1,1.3-1.8l23.3-8.2a4.1,4.1,0,0,1,1.8,0l23.5,8.2a1.92,1.92,0,0,1,1.3,1.8l-.1,8.5,11.2,1.9V16a3.22,3.22,0,0,0-2.1-3L37.6.2a2.12,2.12,0,0,0-1.8,0L2.1,13A3.22,3.22,0,0,0,0,16V36.5A3.22,3.22,0,0,0,2.7,39.7Z" style="fill:url(#b)"/><g style="opacity:0.15;mix-blend-mode:multiply"><path d="M70.8,58.6,36.85,52.91V64.29h0l23.9,4a2,2,0,0,1,1.6,1.9V76a1.92,1.92,0,0,1-1.3,1.8L37.7,86a2.2,2.2,0,0,1-.8.1h0V98.19h0a2.92,2.92,0,0,0,.9-.2L71.5,85.2a3.22,3.22,0,0,0,2.1-3V61.8A3.51,3.51,0,0,0,70.8,58.6Z" style="fill:url(#c)"/><path d="M73.5,43a3.22,3.22,0,0,0-2.7-3.2L36.85,34v11.4L73.5,51.5Z" style="fill:url(#d)"/><path d="M71.5,13,37.6.2A4.09,4.09,0,0,0,36.85,0V12.2a2.75,2.75,0,0,1,.85.1l23.5,8.2a1.92,1.92,0,0,1,1.3,1.8l-.1,8.5,11.2,1.9V16A3.22,3.22,0,0,0,71.5,13Z" style="fill:url(#e)"/></g></g></svg>
```

Comment:
Id in logo file should be of guid type and xmlns:xlink should not be there in the file.

Please follow the guideline for logo [here](https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md)

Thanks,
Aditi Maheshwari

Reply:
@ericlanteigne-semperis Could ypu please add the sample data in the PR in json format.

Thanks,
Aditi Maheshwari

Reply:
@ericlanteigne-semperis Please move the workbook logo file in and preview images in solution folder you can take reference of folder structure from [here](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/PaloAltoPrismaCloud/Workbooks/Images). and also if its a solution workbook metadata is not needed, so could you please revert the changes of workbook metadata.

Thanks,
Aditi Maheshwari

Reply:
I have done these changes. Please let me know if I have missed anything.

Thank you